src/js/requestAwareRouter.js

```diff
@@ -3,25 +3,33 @@
 //
 // To use this in anger, at a minimum you need to supply `options.path` and `options.requestAwareGrade`.
 //
-// If you would like to pass in additional options to the requestAware grade, add an option like the following:
+// If you would like to pass in additional options to the `gpii.express.handler` grade, add an option like the following:
 //
-// `dynamicComponents: { requestHandler: { custom: "value" } }`
+// `dynamicComponents: { requestHandler: { options: { custom: "value" } } }`
 //
-// Those options will be merged with the defaults as expected.
+// Options that already exist when the router is created will be merged with the defaults as expected.  This approach
+// will not work for dynamic values such as model or member variables.
+//
+// For dynamic values, you will need to use an IoC reference from within your `handler` grade's definition, as in:
+//
+// fluid.defaults("my.handler", {
+//   gradeNames: ["gpii.express.handler"],
+//   foo: "{gpii.express.requestAware.router}.foo
+// }
 
 "use strict";
 var fluid     = fluid || require("infusion");
 var gpii      = fluid.registerNamespace("gpii");
 fluid.registerNamespace("gpii.express.requestAware.router");
 
-gpii.express.requestAware.router.getRouter = function (that) {
+gpii.express.requestAware.router.getHandler = function (that) {
     return function (req, res) {
         that.events.onRequest.fire(req, res);
     };
 };
 
 fluid.defaults("gpii.express.requestAware.router", {
-    gradeNames: ["gpii.express.router", "autoInit"],
+    gradeNames: ["gpii.express.router"],
     timeout: 5000, // The default timeout we will pass to whatever grade we instantiate.
```

Comment:
This default duplicates the one which is in gpii.expres.handler - see comment above about organising this timeout system more clearly.